### PR TITLE
feat auto run xattr when minikube gui update

### PR DESCRIPTION
**Pull request by aiyijing**
Close https://github.com/kubernetes-sigs/minikube-gui/issues/79

### CODE REVIEW

**Comment by aiyijing** [comment]
To test: 
**Step 1**: You can change the windows.cpp version to 0.1.0. because https://storage.googleapis.com/minikube-gui/releases.json has not been updated yet. and then run `rm -fr ~/.minikube-gui/last_update_check`

**Step 2**:  run minikube gui
image

**Step 3**:  ok to download new minikube gui and click restart
image

**Step 4**:  Application folder will open and minikube gui restart
image

File: src/updater.cpp [hunk 1]
```diff
@@ -151,10 +151,12 @@ void Updater::downloadUpdate(QString link)
...
                    { "-c",
-                     "cd $TMPDIR && rm -rf /Applications/minikube-gui.app/ && mv "
-                     "minikube-gui-macos/minikube-gui.app/ /Applications/ && rm -rf "
-                     "minikube-gui-macos && open -R /Applications/minikube-gui.app/ && open "
-                     "/Applications/minikube-gui.app/" });
+                     "cd $TMPDIR && rm -rf /Applications/minikube-gui.app/ &&"
+                     "mv minikube-gui-macos/bin/minikube-gui.app /Applications/ &&"
+                     "rm -rf minikube-gui-macos && "
+                     "xattr -d com.apple.quarantine /Applications/minikube-gui.app; true &&"
+                     "open -R /Applications/minikube-gui.app/ && "
```
Comment (medyagh) [inline src/updater.cpp:158]: we can get rid of open -R that was meant to open the finder so the user manually opens the App
now that we xattr it we dont need to do that anymore !

File: src/updater.cpp [hunk 2]
```diff
@@ -135,9 +135,9 @@ void Updater::downloadUpdate(QString link)
...
     QProcess *process = new QProcess(this);
-    QString cmd = QString("cd $TMPDIR && curl -LO %1 && tar -xf minikube-gui-macos.tar.gz && rm "
-                          "minikube-gui-macos.tar.gz")
-                          .arg(link);
+    QString cmd = QString("cd $TMPDIR && curl -LO %1 && mkdir -p minikube-gui-macos && "
+                          "tar -xf minikube-gui-macos.tar.gz -C minikube-gui-macos && "
+                          "rm minikube-gui-macos.tar.gz").arg(link);
     process->start("bash", { "-c", cmd });
     process->waitForFinished(-1);
     QDialogButtonBox buttonBox(Qt::Horizontal, &dialog);
```
Comment (medyagh) [inline src/updater.cpp:143]: in line 147 we can get rid of Step 1 and 2

```
   "Download complete!\n\nAfter clicking 'Restart' do the following:\n\n1. You will see: "
            "\"minikube-gui cannot be opened...\" click 'Cancel'\n2. Your Appliciations folder "
            "will be opened, right click on minikube-gui and click 'Open'")
```

and also for the Button that says restart, we can name it, "re-oepn gui " so ppl dont think it restarts actual minikube

**Review by medyagh — CHANGES_REQUESTED** [review]

**Comment by aiyijing** [comment]
😄 Thanks for you advise,  i have modified it and updated the test result

**Review by medyagh — APPROVED** [review]

**Comment by medyagh** [comment]
thank you @aiyijing

**Pull request merged**